pyqtgraph/icons/__init__.py

```diff
@@ -0,0 +1,37 @@
+import os.path as op
+
+from ..Qt import QtGui
+
+__all__ = ['get_graph_icon']
+
+_ICON_REGISTRY = {}
+
+
+class GraphIcon:
+    """An icon place holder for lazy loading of QIcons"""
+
+    def __init__(self, path):
+        self._path = path
+        self._icon = None
+        name = path.split('.')[0]
+        _ICON_REGISTRY[name] = self
+
+    @property
+    def qicon(self):
+        if self._icon is None:
+            self._icon = QtGui.QIcon(op.join(op.dirname(__file__), self._path))
+
+        return self._icon
+
+
+def get_graph_icon(name):
```

Comment:
probably should use `getGraphIcon` to match camelCase convention elsewhere

Reply:
done